index.html

```diff
@@ -2175,28 +2154,33 @@ <h2>Sessions</h2>
  an <a>intermediary node</a> will also <a>close the session</a> of
  the <a>associated session</a>.
 
-<p>All <a>commands</a>, except <a>New Session</a> and <a>Status</a>,
- have an associated <dfn>current session</dfn>,
- which is the <a>session</a> in which that <a>command</a> will run.
+<p>A <a>session</a> has a <dfn>session ID</dfn>, which is the string
+ representation of a <a>UUID</a> used to uniquely identify the
+ session. This is set when creating the session.
+
+<p>A <a>session</a> has a <dfn>HTTP flag</dfn> which is set when the
+session is created. A session with this flag set is a <dfn>HTTP
+session</dfn>.
+
+<p>A <a>session</a> has an associated list of <a>active input sources</a>.
+
+<p>A <a>session</a> has an associated <a>input state table</a>.
+
+<p>A <a>session</a> has an associated <a>input cancel list</a>.
 
 <p>A <a>remote end</a> has an associated list of
- <dfn data-lt="active session">active sessions</dfn>,
- which is a list of all <a>sessions</a> that are currently started.
- A <a>remote end</a> that is not an <a>intermediary node</a>
- has at most one <a>active session</a> at a given time.
-
-<p>A <a>remote end</a> has an associated
- <dfn>maximum active sessions</dfn> (an integer)
- that defines the number of <a>active sessions</a>
- that are supported.
- This may be “unlimited” for <a>intermediary nodes</a>,
- but must be exactly one for a <a>remote end</a> that is an <a>endpoint node</a>.
-
-<p>A <a>session</a> has an associated <dfn>session ID</dfn> (a string
- representation of a <a>UUID</a>) used to uniquely identify this
- session.  Unless stated otherwise it is <a><code>null</code></a>.
-
-<p>A <a>session</a> has an associated <dfn>current browsing
+ <dfn>active sessions</dfn>, which is a list of all <a>session</a>s
+ that are currently started.
+
+<p>A <a>remote end</a> has an associated list of
+ <dfn>active HTTP sessions</dfn>, which is a list of all <a>HTTP
+ session</a>s that are currently started.
+
+<p class=note>The limitation of a single HTTP session for <a>endpoint
+node</a>s means that the first entry in the list of <a>active HTTP
+sessions</a> will be the only entry.
```

Comment:
Given that we allow a single active HTTP session only do we actually have to carry a list of HTTP sessions?